rest-api/etl/etl.sql

```diff
@@ -1941,50 +1906,19 @@ WHERE tmp1.systolic_blood_pressure_ind != 0              -- take only systolic b
     AND tmp2.diastolic_blood_pressure_ind != 0             -- take only diastolic blood pressure measurements
 ;
 
--- ---------------------------------------------------------------------
--- unit: meas.meas1 - to link parent measurements and child measurements
--- ---------------------------------------------------------------------
-DROP TABLE IF EXISTS cdm.tmp_fact_rel_mm;
-
--- ---------------------------------------------------------------------
--- tmp_fact_rel_mm contains child-to-parent measurements relations
--- ---------------------------------------------------------------------
-CREATE TABLE cdm.tmp_fact_rel_mm AS 
-SELECT
-    cdm_meas1.measurement_id         AS measurement_id_child,
-    cdm_meas2.measurement_id         AS measurement_id_parent
-FROM cdm.measurement cdm_meas1
-INNER JOIN cdm.measurement cdm_meas2
-    ON cdm_meas1.parent_id = cdm_meas2.meas_id
-;
-
 -- ---------------------------------------------------------------------
 -- Insert into fact_relationship child-to-parent measurements relations
 -- ---------------------------------------------------------------------
 INSERT INTO cdm.fact_relationship
 SELECT
     21                              AS domain_concept_id_1,     -- Measurement
-    fr.measurement_id_child         AS fact_id_1,
+    cdm_meas.parent_id              AS fact_id_1,
```

Comment:
These are now reversed `fact_id_1` was child, now its parent. Was that intentional? Table comment seems to indicate child, parent order.

Reply:
No, that was a mistake. Fixed.